ember-client/shared/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -273,17 +273,19 @@ final class EmberClientBuilder[F[_]: Async] private (
               )
             }
           )
-          responseResource <- Resource.makeCase(
-            ClientHelpers
-              .request[F](
-                request,
-                managed.value,
-                chunkSize,
-                maxResponseHeaderSize,
-                idleConnectionTime,
-                timeout,
-                userAgent,
-              )
+          responseResource <- Resource.makeCaseFull((poll: Poll[F]) =>
+            poll(
+              ClientHelpers
+                .request[F](
```

Comment:
FWIW: `ClientHelpers.request` should know how to clean up after itself in case it gets canceled. Though, the release step doesn't care about the cancellation at all (`Applicative[F].unit`).

Reply:
Yeah, it only does something when successful. The idea being any errors or cancellation meaning the connection is going to get scrapped.